### ISSUE TITLE
Format für Dateibenennungen

### ISSUE THREAD

**Issue by sieversMartin** [dante-ev/Vortraege_Tagungen#3]
Ich habe damit begonnen, die vorhandenen PDF-Dateien aus der alten Website ins Repository einzupflegen. Bevor ich das hochlade, würde ich gerne eine einheitliche Benennung abstimmen.

Habt ihr Vorschläge? Eher Autorenname oder Titel oder beides? Tagungsnummer integrieren?

Beispiele:

- Sievers_Herbst2016
- MartinSievers_Herbst2016
- Soziale_Medien_mit_biblatex_zitieren
- Sievers-Soziale_Medien_mit_biblatex_zitieren
- ...

**Comment by UweZiegenhagen** [comment]
Da gab es einen Vorschlag von Oliver 

Von meinem iPhone gesendet

**Comment by koppor** [comment]
Oh, erscheint das "CONTRIBUTING.md" File nicht prominent? --> https://github.com/dante-ev/Vortraege_Tagungen/blob/master/CONTRIBUTING.md?

Vorschlag war:

Name the file following the pattern `year-[Frühjahr|Herbst]/firstname lastname - titleofyourtalk - [dante]year-[fruehjar|herbst].pdf`  
   Example: `2019-Herbst/Oliver Kopp - LaTeX Template Generator - dante2019-herbst.pdf`

**Comment by sieversMartin** [comment]
Offenbar vor lauter Verzeichnissen untergegangen ...

Ihr möchtet also wirklich Leerzeichen im Dateinamen?

**Comment by sieversMartin** [comment]
Gemäß der Benennung wäre ich im übrigen für eine Umbenennung der Verzeichnisse von `Frühling` in `Frühjahr`.

**Comment by UweZiegenhagen** [comment]
Klingt sinnvoll 

Von meinem iPhone gesendet

**Comment by koppor** [comment]
Klar. Direkt machen oder PR schicken. - Seiteneffekte gibt's wohl noch
keine...

Habe noch keinen guten Inhaltsverzeichnisgenrator gefunden -
https://github.com/koppor/gitbook-summary-generator ließe sich dafür
schnell umbauen...

Martin Sievers <notifications@github.com> schrieb am Di., 12. Nov. 2019,
10:27:

**Comment by sieversMartin** [comment]
Ich bin immer noch skeptisch, was Leerzeichen und andere Sonderzeichen angeht. Seht ihr keine Probleme?

**Comment by UweZiegenhagen** [comment]
Naja, wir sind im Jahr 2020, sollte keine mehr geben... :-D
